### ISSUE TITLE
API searches with categories only searches lowest # category

### ISSUE THREAD

**Issue by claym** [theotherp/nzbhydra2#300]
Current version: 2.3.4 

All three of these queries return the exact same results:

`https://nzbhydra2/api?t=search&cat=3000&extended=1&apikey=NOPE&offset=0&limit=100&q=Black%20Crowes%20Shake%20Your%20Money%20Maker`

`https://nzbhydra2/api?t=search&cat=3000,3010,3020,3030,3040&extended=1&apikey=NOPE&offset=0&limit=100&q=Black%20Crowes%20Shake%20Your%20Money%20Maker`

`https://nzbhydra2/api?t=search&cat=3010,3000,3020,3030,3040&extended=1&apikey=NOPE&offset=0&limit=100&q=Black%20Crowes%20Shake%20Your%20Money%20Maker`

However, if I search this query, I get a number of results not round in the two queries that 3010 is included in above.

`https://nzbhydra2/api?t=search&cat=3010&extended=1&apikey=NOPE&offset=0&limit=100&q=Black%20Crowes%20Shake%20Your%20Money%20Maker`

Is there a problem with the query method, or is it an issue with category parsing?

**Comment by theotherp** [comment]
You didn't post the debug infos zip as requested.

**Comment by claym** [comment]
[nzbhydra-debuginfos-2019-02-05-09-06.zip](https://github.com/theotherp/nzbhydra2/files/2832367/nzbhydra-debuginfos-2019-02-05-09-06.zip)
Log files attached!

**Comment by theotherp** [comment]
When you search using the categories 3000 the category "Audio" is automatically applied. The preconfigured required words for this category are "MP3" and "FLAC" (case insensitive). That doesn't make sense because both must be present which is rarely the case. No idea why I did that, it's just a stupid bug.

When you search using 3010 the audio category is not automatically applied.

When you search for 3000, 3010, 3020, 3030, 3040 the audio category is applied and only 3000 is used because 3000 includes 3010, 3020, 3030, 3040.

Long story short, I'll remove the required words from the audio category. But that will only affect new installations. I recommend you remove them in your config and see if that helps.

**Comment by claym** [comment]
Yep, looks like that fixed it, removing MP3/FLAC required tags from the Audio category.
